ansible/roles/ocp4-workload-camelk-lab/defaults/main.yml

```diff
@@ -2,7 +2,9 @@ labguide_project_name: labguide
 #labguide_project_display_name: CamelK Lab Guide
 
 console_url: https://console-openshift-console.{{subdomain_base}}
-api_url: "{{master_public_url}}"
+api_url: https://api.cluster-{{guid}}.{{guid}}.events.opentlc.com:6443
```

Comment:
You don't want to hardcode events.opentlc.com here.
Use `{{ subdomain_base }}`

Reply:
Done